modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -547,6 +576,17 @@ ONNXCompiled::ONNXCompiled(const gapi::onnx::detail::ParamDesc &pp)
         }
     }
 
+    if (!params.const_inputs.empty()) {
+        // Form input names order without const input names
+        in_names_without_const.clear();
+        std::copy_if(params.input_names.begin(), params.input_names.end(),
+                     std::back_inserter(in_names_without_const),
+                     [&](std::string& name) {
```

Comment:
I believe it can be `const std::string&`

Reply:
Fixed.